src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+
+    private double height;
+
+    public RightTriangle(Colors color, double side, double height) {
+        super(color, side);
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle, area: "
+                + getArea() + "sq. units, side "
+                + getSide() + " units, height " + height
+                + " units, color: " + getColor().name());
+    }
+
+    @Override
+    public double getArea() {
+        return getSide() * 0.5 * height;
```

Comment:
All magic numbers in your code should be constants. In this case, the number '0.5' should be extracted as a constant to give it a meaningful name and make the code more readable.